README.md

```diff
@@ -94,44 +94,18 @@ This repository has a baseline structure for the project that includes several f
 
```

Comment:
It's above this change, but just wanted to mention that the solution/obfuscated branch mentioned in the running the game part no longer works. It produces a number of error messages when trying to `npm install`, and even if we `npm install` in a working branch and then switch over, it has module loading errors. Maybe we should excise mention of that branch?

Reply:
👍